Cargo.toml

```diff
@@ -13,3 +13,4 @@ path = "src/mirkat.rs"
 
 [dependencies]
 byteorder = "1"
+extprim = "1"
```

Comment:
Do you need extprim if you're turning on nightly's support u128?

Reply:
Ah maybe, I did this because of a compiler error when using `u128`. Maybe I just need to enable the feature only.

Reply:
OK, let me know.

Reply:
You're right. I don't need this. Removed